spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -1,190 +1,138 @@
         get :show, id: model
       end
 
-      it_behaves_like 'GET show success'
-      it 'should include @pending reservations' do
-        # Make one overdue reservations, one active reservation that started
-        # in the past, one active reservation starting today,
-        # one reservation starting within the next week,
-        # and one starting more than a week in the future
-        #
-        #
-        # First, make sure we have enough equipment objects available
-        FactoryGirl.create(:equipment_item, equipment_model: model)
-        FactoryGirl.create(:equipment_item, equipment_model: model)
-        FactoryGirl.create(:missed_reservation, equipment_model: model)
-        res_starting_today =
-          FactoryGirl.create(:reservation, equipment_model: model,
-                                           start_date: Time.zone.today,
-                                           due_date: Time.zone.today + 2.days)
-        res_starting_this_week =
-          FactoryGirl.create(:reservation, equipment_model: model,
-                                           start_date: Time.zone.today + 2.days,
-                                           due_date: Time.zone.today + 4.days)
+      it_behaves_like 'success', :show
+      it 'sets to correct equipment model' do
+        expect(assigns(:equipment_model)).to eq(model)
+      end
+      it 'sets @associated_equipment_models' do
+        mod1 = FactoryGirl.create(:equipment_model)
+        model.associated_equipment_models = [mod1]
+        get :show, id: model
+        expect(assigns(:associated_equipment_models)).to eq([mod1])
+      end
 
-        FactoryGirl.create(:reservation, equipment_model: model,
-                                         start_date: Time.zone.today + 10.days,
-                                         due_date: Time.zone.today + 12.days)
+      it 'limits @associated_equipment_models to maximum 6' do
+        model.associated_equipment_models =
+          FactoryGirl.create_list(:equipment_model, 7)
         get :show, id: model
-        expect(assigns(:pending)).to eq([] << res_starting_today <<
-                                            res_starting_this_week)
+        expect(assigns(:associated_equipment_models).size).to eq(6)
       end
     end
+    USER_ROLES.each { |type| it_behaves_like 'GET show success', type }
 
-    context 'with non-admin user' do
+    context 'with admin user' do
       before do
-        sign_in FactoryGirl.create(:user)
+        FactoryGirl.create_pair(:equipment_item, equipment_model: model)
+        sign_in FactoryGirl.create(:admin)
+      end
+      let!(:missed) do
+        FactoryGirl.create(:missed_reservation, equipment_model: model)
+      end
+      let!(:starts_today) do
+        FactoryGirl.create(:reservation, equipment_model: model,
+                                         start_date: Time.zone.today,
+                                         due_date: Time.zone.today + 2.days)
```

Comment:
the dates here could be deleted but I think that because we are explicitly relying on these dates for the test to pass they should stay